include/swift/ABI/TaskStatus.h

```diff
@@ -319,6 +292,99 @@ class EscalationNotificationStatusRecord : public TaskStatusRecord {
   }
 };
 
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+JobPriority
+swift_task_escalate(AsyncTask *task, JobPriority newPriority);
```

Comment:
Yea this is really gross but I can't include Runtime/Concurrency.h here which is where the function is declared, and I really just want to forward declare this function so that the `performEscalationAction` can invoke it. 

Is there a better way to do this?

Reply:
Ah I just saw your comment below about defining the method out-of-line in the runtime. Let me try that.